packages/map-template/src/components/Search/Search.jsx

```diff
@@ -154,6 +163,10 @@ function Search({ onSetSize }) {
         } else {
             searchFieldRef.current.focusInput();
         }
+
+        if (showKeyboard) {
```

Comment:
Is this necessary? Couldn't we just use the Recoil value in the JSX?

Reply:
There was a missing function call here for the `getInputField` on the search ref component 😄

Reply:
But I guess we could get rid of the useState `isKeyboardVisible` and just use the Recoil value